tests/e2e/core-tests/specs/shopper/front-end-checkout-login-account.test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, jest/expect-expect */
+/**
+ * Internal dependencies
+ */
+ const {
+	shopper,
+	merchant,
+	createSimpleProduct,
+	uiUnblocked,
+	setCheckbox,
+	settingsPageSaveChanges,
+} = require( '@woocommerce/e2e-utils' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe,
+	beforeAll,
+} = require( '@jest/globals' );
+
+const config = require('config');
+const simpleProductName = config.get('products.simple.name');
+
+const runCheckoutLoginAccountTest = () => {
+	describe('Shopper Checkout Login Account', () => {
+		beforeAll(async () => {
+			await merchant.login();
+			await createSimpleProduct();
+
+			// Set checkbox for logging to account during checkout
+			await merchant.openSettings('account');
+			await setCheckbox('#woocommerce_enable_checkout_login_reminder');
+			await settingsPageSaveChanges();
+			await merchant.logout();
+
+			// Add simple product to cart and proceed to checkout
+			await shopper.goToShop();
+			await shopper.addToCartFromShopPage(simpleProductName);
+			await uiUnblocked();
+			await shopper.goToCheckout();
+		});
+
+		it('can login to an existing account during checkout', async () => {
+			// Click to login during checkout
+			await page.waitForSelector('.woocommerce-form-login-toggle');
+			await expect(page).toClick('.woocommerce-info > a.showlogin');
+
+			// Fill shopper's login credentials and proceed further
+			await page.type( '#username', config.get('users.customer.username') );
+			await page.type( '#password', config.get('users.customer.password') );
+
+			await Promise.all([
+			page.waitForNavigation({waitUntil: 'networkidle0'}),
+			page.click('button[name="login"]'),
+		]);
+
+			// Place an order
+			await shopper.placeOrder();
+			await expect(page).toMatchElement('h1.entry-title', {text: 'Order received'});
+
+			// Verify the email of a logged in user
+			await expect(page).toMatchElement('ul > li.email', {text: 'Email: john.doe@example.com'});
+
+			// Verify the user is logged in on my account page
+			await shopper.gotoMyAccount();
+			expect(page.url()).toMatch('my-account/');
```

Comment:
I noticed this is missing an `await`, just wanted to check if this was intended here to help with the test running. Everything passed fine for me, but wanted to verify.

Reply:
Hey, yeah.. I didn't add it because that type of an expression doesn't need `await`. But in order to be in line with the rest and more elegant in design, I'll add it. 👍

Reply:
Sounds good, thanks!